docs/shared/getting-started/intro.md

```diff
@@ -1,100 +1,87 @@
+- **Automate Dependency Update**: if you leverage Nx plugins you gain additional features such as [code generation](/core-features/plugin-features/use-code-generators) and tools to [automatically upgrade](core-features/automate-updating-dependencies) your codebase and dependencies.
+- **Make it Your Own**: Nx is highly customizable and extensible. Fine-tune it by [create your own plugins](/extending-nx/intro/getting-started) and optionally even [share them with the community](/extending-nx/tutorials/publish-plugin#publish-your-nx-plugin).
 
-```shell
-npx create-nx-workspace
-```
+<!-- - **Monorepo and Single Projects**: Nx supports both, monorepos as well as single-project (standalone) workspaces. -->
 
-{% /tab %}
-{% tab label="yarn" %}
-
-```shell
-npx create-nx-workspace --pm yarn
-```
+Find out more about [why you should use Nx](/getting-started/why-nx) or browse our [core features](/core-features) section.
 
-{% /tab %}
-{% tab label="pnpm" %}
+## Try Yourself!
 
 ```shell
-npx create-nx-workspace --pm pnpm
+npx create-nx-workspace@latest
 ```
 
-{% /tab %}
-{% /tabs %}
-
-You can use Nx to quickly scaffold a new project or even an entire monorepo. It can be incrementally adopted and will grow with your project as it scales.
+## Learn Nx
 
-{% cards cols="3" %}
+{% cards cols="4" %}
 
-{% title-card title="New Monorepo" url="#get-started-with-the-basics" /%}
-{% title-card title="Choose a Stack" url="#learn-about-nx-and-your-favorite-stack" /%}
-{% title-card title="Add to an Existing Project" url="#adding-nx-to-an-existing-project" /%}
+{% link-card title="Nx in 5 minutes!" type="video" url="/getting-started/tutorials/package-based-repo-tutorial" icon="/documentation/shared/images/icons/nx-color.svg" /%}
 
-{% /cards %}
+{% link-card title="Create a Package-based Monorepo" type="tutorial" url="/getting-started/tutorials/package-based-repo-tutorial" icon="/documentation/shared/images/icons/javascript-color.svg" /%}
 
-## Get Started with the Basics
+{% link-card title="Create a Monorepo with Nx Plugins" type="tutorial" url="/getting-started/tutorials/integrated-repo-tutorial" icon="/documentation/shared/images/icons/javascript-color.svg" /%}
 
-Its modular architecture lets you adopt Nx for package-based monorepos in combination with NPM, Yarn or PNPM, or create a fully integrated monorepo using Nx plugins. Learn more with the tutorials below.
+{% link-card title="On our YT Channel" type="video" url="https://www.youtube.com/@nxdevtools" icon="/documentation/shared/images/icons/youtube-color.svg" /%}
 
-{% personas %}
-{% persona type="javascript" title="New Package-Based Repo" url="/getting-started/tutorials/package-based-repo-tutorial" %}
-Create a monorepo with Yarn, NPM or PNPM. Nx makes it fast, but lets you run things your way.
+{% /cards %}
 
-- [Get started with your package-based repo](/getting-started/tutorials/package-based-repo-tutorial)
+## Know Already? Pick Your Stack!
```

Comment:
```suggestion
## Pick Your Stack!
```

Reply:
"Know Already?" has no clear object in the sentence.  If you're trying to ask if they know Nx already, that's already implied by the section above being "Learn Nx".  Another option is to number the headers: 1. Learn Nx.  2. Pick Your Stack!

I also don't want to have "Know Nx Already?" on the intro page, because nobody can self-diagnose if they know Nx.  They know some things about it, but how much are you asking them to know?

Reply:
This "Know already" was more meant in the direction of "Know already what you want", like which stack so go ahead and pick one. Something like that